boards/xtensa/intel_s1000_crb/intel_s1000_crb.dts

```diff
@@ -31,3 +31,14 @@
 		label = "TLV320DAC";
 	};
 };
+
+&spi0 {
+	status = "ok";
+
+	mx25um512@0 {
+		compatible = "spi,flash";
+		label = "MX25UM512";
+		reg = <0x00>;
+		spi-max-frequency = <2000000>;
```

Comment:
Since this is max frequency, it should be 100MHz. The actual operating frequency (if there is a variable for that) can be 2MHz.

Reply:
I don't think we have any such variable in dts.